.github/workflows/ci_tests.yml

```diff
@@ -56,6 +56,14 @@ jobs:
           distribution: temurin
           cache: gradle
 
+      - name: Pull genai-common
+        if: matrix.module == ':firebase-vertexai'
+        run: |
+          git clone https://github.com/google-gemini/generative-ai-android.git
+          cd generative-ai-android
+          ./gradlew :common:publishToMavenLocal
```

Comment:
If the version in common is no the same as the version in `firebase-vertexai` then it will not pull the newly created artifact. If you can change the version to something like `9999`, publish that, and then make `firebase-vertexai` depend on that version, we are absolutely sure it came from the just cloned repo.